README.md

```diff
@@ -211,14 +269,14 @@ You can find an example of the import performance to expect with this image on t
 
 If you encounter such entries in the log, it will mean that the default shared memory limit (64 MB) is too low for the container and it should be raised:
 ```
-renderd[121]: ERROR: failed to render TILE ajt 2 0-3 0-3
+renderd[121]: ERROR: failed to render TILE default 2 0-3 0-3
```

Comment:
Passt hier der map-style? "ajt" War früher der Name. Ist das jetzt default?

Reply:
Nach Punkt 2 in https://github.com/Overv/openstreetmap-tile-server/pull/234#issuecomment-1169457142 sieht das so aus.